ls1intum/Artemis
# [Feature] Cypress end to end testing setup

### Checklist

### Motivation and Context
 We need e2e

### Description
 Added cypress setup

### Steps for Testing


1. Start local artemis server at :8080 or configure server target in cypress.json('baseUrl')
2. Configure admin/user account password in Artemis/cypress.env.json (currently only "username", "password", "adminUsername" and "adminPassword" is needed)
3. In artemis project in terminal enter:
4. yarn cypress:open (to open the cypress testing browser)
5. yarn cypress:run (to run the existing cypress tests in CLI)
6. Check that the tests pass in the [new E2E pipeline](https://bamboo.ase.in.tum.de/browse/ARTEMIS-AETBB)
